Examples/Applications/GeometryViewer/index.js

```diff
@@ -360,9 +361,11 @@ function createPipeline(fileName, fileContents) {
   renderer.addActor(actor);
 
   // Manage update when lookupTable change
-  lookupTable.onModified(() => {
-    renderWindow.render();
-  });
+  // too heavy handed, causes multiple buffer pushes as opposed
+  // to defering them to when all changes are done
+  // lookupTable.onModified(() => {
+  //   renderWindow.render();
+  // });
```

Comment:
We should register the listener with a priority of -1 and debounce the call to render.